ibis_mssql/tests/test_ops.py

```diff
@@ -0,0 +1,26 @@
+from ibis.tests.expr.mocks import MockConnection
+import unittest
+import pytest
+
+pytest.importorskip('sqlalchemy')
+
+
+class TestOps(unittest.TestCase):
+    def setUp(self):
+        self.con = MockConnection()
+
+    def test_join_column_count(self):
+        t1 = self.con.table('star1')
+        t2 = self.con.table('star2')
+
+        # msc = MSSQLClient()
+        # new_t1 = msc.database(name=t1) <-- use instances of ibis_mssql tables
+        # new_t2 = msc.database(name=t2) <-- use instances of ibis_mssql tables
+
+        predicate = t1['foo_id'] == t2['foo_id']
+
+        joined = t1.inner_join(t2, predicate)[
```

Comment:
thanks for adding this test.

I think that other way to do this would be something like:

```python

def test_join_column_count(con, alltypes):
  t1 = alltypes
  t2 = alltypes

  t1 = t1[t1.index, t1.field1, t1.field2]
  t2 = t2[t2.index, t2.field3, t2.field4]

  ...

```

where field1..4 are placeholders

Reply:
Working on this ... having some DB connection issues to get over.

Reply:
feel free to share the errors you have if you want any help :)